src/main/java/io/cdap/e2e/pages/actions/CdfSignInActions.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.actions;
+
+import io.cdap.e2e.pages.locators.CdfSignInLocator;
+import io.cdap.e2e.utils.ConstantsUtil;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.e2e.utils.WaitHelper;
+import java.io.IOException;
+import static io.cdap.e2e.utils.ConstantsUtil.CDFPASSWORD;
+import static io.cdap.e2e.utils.ConstantsUtil.CDFUSERNAME;
```

Comment:
unless there is a specific usecase, we should not go for static imports, it makes the code unreadable.

Please use `ConstantsUtil.CDFPASSWORD` and so on wherever required.

Reply:
@Vipinofficial11 can you address comment so that we can get to a closure sooner.

Reply:
@itsankit-google updated [36bc9cff025b14bc9e7b456af4d9fa4b87c42082]